src/WooCommerceCheckout.php

```diff
@@ -118,4 +125,29 @@ public static function checkConfirmationEmailField() {
     }
   }
 
+  /**
+   * Check if we are using Amazon V2 in the checkout.
+   *
+   * @return bool
+   */
+  public static function isAmazonPayV2Checkout(): bool {
+    $is_amzon_pay_active = is_plugin_active('woocommerce-gateway-amazon-payments-advanced/woocommerce-gateway-amazon-payments-advanced.php') ?: FALSE;
+    if ($is_amzon_pay_active && isset(WC()->session)) {
```

Comment:
1. is_plugin_active() only returns true or false, so the ternary is unnecessary.
2. Typo in "amzon".
```suggestion
    $is_amazon_pay_active = is_plugin_active('woocommerce-gateway-amazon-payments-advanced/woocommerce-gateway-amazon-payments-advanced.php');
    if ($is_amazon_pay_active && isset(WC()->session)) {
```